.github/workflows/release.yml

```diff
@@ -9,7 +9,6 @@ on:
 
 env:
   NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: fe319abf1a1b0d76b39894e5e17aff0c
-  NEXT_PUBLIC_WALLET_PROJECT_ID: 768afa89687ff9f4d0ec5a643e4ace29
```

Comment:
No need to create a new env var for this, just use the existing `NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID`. We can set it to different values per project deployment if we want.